internal/backend/storage/gitfs/ssh_others.go

```diff
@@ -0,0 +1,14 @@
+// +build !windows
+
+package gitfs
+
+// gitSSHCommand returns a SSH command instructing git to use SSH
+// with persistent connections through a custom socket.
+// See https://linux.die.net/man/5/ssh_config and
+// https://git-scm.com/docs/git-config#Documentation/git-config.txt-coresshCommand
+//
+// Note: Setting GIT_SSH_COMMAND, possibly to an empty string, will take
```

Comment:
This should be documented in the docs, no?

Reply:
Yes, indeed